rgw/v2/tests/s3_swift/configs/test_lc_date.yaml

```diff
@@ -0,0 +1,22 @@
+# script: test_bucket_lifecycle_config_ops.py
```

Comment:
does this YAML file run with the new script that you have written,? if so, can you change it to the correct script name in this comment section check the same for the all the new YAML files in this PR

Reply:
Done. I have added the script name for all the yamls to "#test_object_expiration.py'